week-2/I-array-properties/exercise.js

```diff
@@ -6,9 +6,12 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  if (arr.length == 0) {
```

Comment:
How can you simplify this code? `isEmpty` is meant to return a boolean, which it does at the moment, but `arr.length == 0` on its own also returns a boolean! How can you achieve the same without `if` and `else`?

Reply:
we can simplify as follows: 
function isEmpty(arr) {
  return arr.size();
}